developer-docs/latest/deployment/heroku.md

```diff
@@ -399,6 +399,20 @@ git commit -m "Update database config"
 git push heroku master
 ```
 
+::: notice
```

Comment:
Can you please just include the options in the documentation step instead of in a notice.